uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_forms.scss

```diff
@@ -5,6 +5,19 @@ form {
     padding-right: 0;
   }
 
+  /* Bootstrap override */
+  .input-group-addon {
```

Comment:
Scope of this style shouldn't be limited only to `.form-horizontal` but all forms inside Tasks&Assignments.